docs/sources/linking/data-link-variables.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "URL variables"
+keywords = ["grafana", "url variables", "documentation", "variables", "data link"]
+aliases = ["/docs/grafana/latest/variables/url-variables.md","/docs/grafana/latest/variables/variable-types/url-variables.md"]
+type = "docs"
+weight = 400
++++
+
+# Data link variables
+
+You can use certain variables in data links to link to specific portions of your visualizations. For more information about data links, refer to [Data links]({{< relref "data-links.md" >}}).
+
+To see a list of available variables, type `$` in the data link **URL** field to see a list of variables that you can use.
+
+> **Note:** These variables changed in 6.4 so if you have an older version of Grafana, then use the version picker to select docs for an older version of Grafana.
+
+For more information about using variables, refer to [Templates and variables]({{< relref "../variables/templates-and-variables.md" >}})
+
+## Time range panel variables
+
+These variables allow you to set the time range for a dashboard. For more information about time settings and options in Grafana, refer to [Time range controls]({{< relref "../dashboards/time-range-controls.md" >}})
+
+* ``__url_time_range`` - current dashboard's time range (i.e. ``?from=now-6h&to=now``)
```

Comment:
we sometimes only refer to variables using __varName and not the full `${varName}` syntax, wonder if we should always do `${varName} everywhere here as that is what you can copy paste into Grafana

Reply:
As a user, I would find that super helpful. I like just copying and pasting things from help docs into my terminal. Makes life much easier.

I suggest we do that in a different PR, and I would want some help evaluating whether there are some instances (I'm thinking data source docs, mostly) where we wouldn't want to apply that format.